Civi/API/Subscriber/SiteEmailLegacyOptionValueAdapter.php

```diff
@@ -201,10 +201,6 @@ private function preprocessApi4SiteEmailOptionValues(AbstractAction $apiRequest)
     $entityNameProperty = $reflection->getProperty('_entityName');
     $entityNameProperty->setAccessible(TRUE);
     $entityNameProperty->setValue($apiRequest, 'SiteEmailAddress');
-    // Also reset $_entityFields
-    $entityFieldsProperty = $reflection->getProperty('_entityFields');
-    $entityFieldsProperty->setAccessible(TRUE);
-    $entityFieldsProperty->setValue($apiRequest, NULL);
```

Comment:
With the changes to entityFields caching being moved out of the class itself, this is no longer necessary. The above lines changing `_entityName` will have the same effect since `entityFields` now keys its cache off of that.